### PR TITLE
Add run_info and timestamps to wptreport

### PR DESCRIPTION
**Pull request by Hexcles**
The first stab at #10511 , adding the simplest fields to make sure I understand correctly @jgraham 

Meanwhile, how shall I pass the version of the browser? Via `run_info`? FWIW, there's `run_info['version']` which is the version of my OS (and it seems to overlap with `run_info['os_version']` a bit).

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/10811)

### CODE REVIEW

File: tools/wptrunner/wptrunner/formatters.py [hunk 1]
```diff
@@ -8,15 +8,21 @@ class WptreportFormatter(BaseFormatter):
 
     def __init__(self):
         self.raw_results = {}
+        self.results = {}
+
+    def suite_start(self, data):
+        self.results['browser'] = data['run_info']['product']
```
Comment (jgraham) [inline tools/wptrunner/wptrunner/formatters.py]: I would be tempted to just dump `run_info` directly into the results.

Comment (Hexcles) [reply tools/wptrunner/wptrunner/formatters.py]: Yeah let's do that.

Meanwhile, could you answer the question I had regarding these fields in `run_info`?
```
"linux_distro": "Debian",
"os": "linux",
"os_version": "sid",
"version": "Debian sid",
```
They seem to present an overlapping set of information. Is this intended? Notably, what's `version`? Is it intended to be the version of the product or the OS?

**Comment by jugglinmike** [comment]
Since we're interested in building on this data from external projects, what do you think about documenting this format in the `docts/` directory?

**Comment by Hexcles** [comment]
@jugglinmike First of all, I'm going to document some of the mandatory fields in my [design doc](https://docs.google.com/document/d/1F3GWJ6lA8C3S5WgyJgXxmTFzu92mqF-1ddGije67ork/edit), including the browser info and revision, which will solve the most pressing issue for uploading results.

Later we can more formally document the whole wptreport here as a doc and/or JSON schema.

File: tools/wptrunner/wptrunner/formatters.py (hunk 1 above)
Comment (jgraham) [reply tools/wptrunner/wptrunner/formatters.py]: All I know is what I can figure out from `https://searchfox.org/mozilla-central/source/testing/mozbase/mozinfo/mozinfo/mozinfo.py`. So `version` is what you get from `uname` and `os_version` has a more custom  implementation.

**Review by jgraham — APPROVED** [review]

**Pull request merged**